chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,41 @@
+---
+title: Content Processing
+---
+
 ### Content Processing
 
-Owner: Vlad Năstase
-Reviewer: Teo Duțu
+Content organized in a Version Control System like `git`, and published to a `GitHub` repository is suitable for easier contributions.
+In order to deliver it to the end user/learner, this content has to be processed and rendered in an easier to navigate form.
```

Comment:
```suggestion
As a consequence, the content should be presented in a much more friendly and interactive manner.
To that end, the content is processed so that it is deliverable to a publisher (website) in a much more approachable way.
```